multimodule-test/accumulo-elasticsearch/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.vertexium</groupId>
+        <artifactId>vertexium-multimodule-test</artifactId>
+        <version>2.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>vertexium-accumulo-elasticsearch-multimodule-test</artifactId>
+    <name>Vertexium: Multi-module tests: Accumulo and Elasticsearch</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.vertexium</groupId>
+            <artifactId>vertexium-test</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.vertexium</groupId>
+            <artifactId>vertexium-accumulo</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
I'm pretty sure it doesn't, but I will double check that. This dependency is so that I could get at the AccumuloResource and ElasticsearchResource. I'll review the Travis log and comment again to verify that it doesn't rerun them.